src/main/java/org/altbeacon/beacon/service/GattBeaconTracker.java

```diff
@@ -1,22 +1,18 @@
 package org.altbeacon.beacon.service;
 
-import android.util.Log;
-
 import org.altbeacon.beacon.Beacon;
 
 import java.util.HashMap;
 
 /**
  * Keeps track of beacons that have ever been seen and
- * merges them together in the case of Gatt beacons
+ * merges them together depending on configured beacon parsers
  * Created by dyoung on 5/5/15.
  */
 public class GattBeaconTracker {
```

Comment:
It might be good to rename this something like `ExtraDataBeaconTracker` since it no longer is specific to GattBeacons